### PR TITLE
Fix tests on ARM Linux

### PR DESCRIPTION
**Pull request by petrochenkov**

### CODE REVIEW

File: src/libstd/sys/unix/backtrace/tracing/gcc_s.rs [hunk 1]
```diff
@@ -53,7 +53,7 @@ pub fn unwind_backtrace(frames: &mut [Frame])
     // See libunwind:src/unwind/Backtrace.c for the return values.
     // No, there is no doc.
     match result_unwind {
-        uw::_URC_END_OF_STACK | uw::_URC_FATAL_PHASE1_ERROR => {
+        uw::_URC_END_OF_STACK | uw::_URC_FATAL_PHASE1_ERROR | uw::_URC_FAILURE => {
```
Comment (Yamakaky) [inline src/libstd/sys/unix/backtrace/tracing/gcc_s.rs]: Maybe add a comment for the followers?

Comment (petrochenkov) [reply src/libstd/sys/unix/backtrace/tracing/gcc_s.rs]: I'm not ever sure how to comment this beyond "I don't care what these errors are, but these three need to be ignored for Travis to pass".

**Comment by petrochenkov** [comment]
Added comment.

**Comment by petrochenkov** [comment]
The previous behavior was to ignore *all* unwinding error codes, so there's a chance further testing will reveal some more codes that need to be ignored.

**Pull request merged**